modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -1,30 +1,38 @@
-= Creating an API instance on API Manager
+= Creating an API instance
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager 2.x, you can use instances of an API in multiple environments, and you can have multiple instances in the same environment. You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies, for example to the proxy, and throttling policies and security policies to the basic endpoint.
+In API Manager 2.x, you can use instances of an API in multiple environments. Additionally, you can also have multiple instances in the same environment. 
```

Comment:
Redundant; omit either "Additionally" or "also." You do not need both.

Reply:
Done! Thanks!